MovieAppPetProject/MovieAppPetProject/Login/LoginViewController.swift

```diff
@@ -9,49 +9,38 @@ import UIKit
 
 class LoginViewController: UIViewController {
 
-    let username = "Admin"
-    let password = "DVTPassword"
+    // MARK: - Outlets
+    @IBOutlet private weak var usernameInput: UITextField!
+    @IBOutlet private weak var passwordInput: UITextField!
 
-    @IBOutlet weak var usernameInput: UITextField!
-    @IBOutlet weak var passwordInput: UITextField!
+    // MARK: - Variable
+    private let loginViewModel = LoginViewModel()
 
+    // MARK: - Functions
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        usernameInput.attributedPlaceholder = NSAttributedString(string: "Username", attributes: [NSAttributedString.Key.foregroundColor: UIColor.gray])
+        passwordInput.attributedPlaceholder = NSAttributedString(string: "Password", attributes: [NSAttributedString.Key.foregroundColor: UIColor.gray])
     }
-    func showAlert(alertTitle: String, alertMessage: String) {
 
-        let alert = UIAlertController(title: alertTitle, message: alertMessage, preferredStyle: .alert)
-        self.present(alert, animated: true)
-        DispatchQueue.main.asyncAfter(deadline: .now() + 1.5) {
-            alert.dismiss(animated: true)
+    @IBAction private func loginTapped(_ sender: Any) {
+        let username = usernameInput.text
+        let password = passwordInput.text
+
+        loginViewModel.login(username: username, password: password) { [weak self] success in
```

Comment:
[weak self] success in can be removed when remove closure

Reply:
removed